website/docs/addon_applications_admin.md

```diff
@@ -35,29 +239,46 @@ For instance:
     ]
 }
 ```
+:::
 
-### Adding versions
+</details>
 
-It is possible to add new version for any supported application. There are two ways of doing it.
+### How to update the task types list in the filter settings ?
 
-1. **Add new executable** to an existing application version. This is a good way if you have multiple fully compatible versions of your DCC across the studio. Nuke is a typical example where multiple artists might have different `v#` releases of the same minor Nuke release. For example `12.2v3` and `12.3v6`. When you add both to `12.2` Nuke executables they will be treated the same in AYON and the system will automatically pick the first that it finds on an artist machine when launching. Their order is also the order of their priority when choosing which version to run if multiple are present.
-![settings_applications](assets/settings/settings_addapplication.gif)
+<details><summary>How to update the task types list in the filter settings ?</summary>
 
-2. **Add version** in case you want this version to be selectable individually. This is typically used for bigger releases that might not be fully compatible with previous versions. Keep in mind that if you add the latest version of an Application that is not yet part of the official AYON release, you might run into problems with integration. We test all the new software versions for compatibility and most often, smaller or bigger updates to AYON code are necessary to keep everything running.
-![settings_applications](assets/settings/settings_addappversion.gif)
+Task types list in Studio settings is generated from the default anatomy preset. 
 
-## Tools
+Task types list in Project settings is generated from the project's anatomy. 
+To navigate quickly, click `a+a`.
```

Comment:
Somehow this provided me with less info than I expected:

![image](https://github.com/user-attachments/assets/27956156-be57-4839-9926-580d3ff541ae)

For whatever reason it made it 'hard to read'.
Should it be bullet points maybe? Or **Studio Settings** and **Project Settings** worded in bold to differentiate the two lines?

Reply:
Hope this is better https://github.com/ynput/ayon-documentation/pull/220/commits/ab452014f98efa0970a03f6a27deed767835e5aa